remove call to sim_params variable, which isn't a function input

mattg3004 opened the pull request:
in sim_Ind_MDA function there is a call to "sim_params['timesim']", which isn't input to function. Change this to just call timesim, which is input to function.

Added a function to return prevalence in target age group for surveys

mattg3004 committed:
remove call to sim_params variable, which isn't a function input

mattg3004 committed:
add a function to do survey

tlestang commented on trachoma/trachoma_functions.py:
```diff
@@ -378,6 +378,31 @@ def sim_Ind_MDA(params, Tx_mat, vals, timesim, demog, bet, MDA_times, seed, stat
+
+
+
+def returnSurveyPrev(vals, TestSensitivity, TestSpecificity):
```
Where is/will be this function used?

mattg3004 replied:
This will be used joint with the next commit, where I have added another function to do the same as sim_Ind_MDA, but with a prevalence survey added onto it to assess how many MDAs should be done, or if we should stop MDA

mattg3004 committed:
add a copy of sim_Ind_MDA function

this copied function will be edited to include the surveying we need for endgame project
